src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/WPFValueProvider.cs

```diff
@@ -69,13 +71,16 @@ private async Task<string> GetPropertyValueAsync(string propertyName, IProjectPr
         return string.Empty;
     }
 
-    private static async Task<bool> IsWPFApplicationAsync(IProjectProperties defaultProperties)
+    private async Task<bool> IsWPFApplicationAsync(IProjectProperties defaultProperties)
```

Comment:
The method name is now a bit misleading. Perhaps "IsWpfAndNotWinFormsApplicationAsync" or something like that.

Reply:
Also it would be helpful to include a comment explaining why it is useful to know when it is WPF but not WinForms.